app/error.tsx

```
@@ -0,0 +1,22 @@
+"use client";
+
+interface ErrorProps {
+	error: Error;
+	reset: () => void;
+}
+
+export default function Error({ error, reset }: ErrorProps) {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)